sdk/nextjs/src/pages/bucketing.ts

```diff
@@ -44,7 +43,7 @@ export const getBucketedConfig = async (
         config: {
             ...bucketedConfig,
             sse: {
-                url: await sseURlGetter(sdkKey, config.ably?.apiKey)(),
+                url: config.sse.hostname + config.sse.path,
```

Comment:
I wonder if you should use `URL(config.sse.path, config.sse.hostname)` to better handle for path issues?